content/docs/command-reference/repro.md

```diff
@@ -10,7 +10,8 @@ analyzing dependencies and <abbr>outputs</abbr> of the target stages.
 ```usage
 usage: dvc repro [-h] [-q | -v] [-f] [-s] [-c <path>] [-m] [--dry] [-i]
                  [-p] [-P] [-R] [--no-run-cache] [--force-downstream]
-                 [--no-commit] [--downstream] [targets [targets ...]]
+                 [--no-commit] [--downstream] [--pull]
```

Comment:
I have a concern about this option's name. See https://github.com/iterative/dvc/pull/4538#issuecomment-706409339, thanks.